### ISSUE TITLE
Add support for another operating system: jupyter-notebook on Binder.org

### ISSUE THREAD

**Issue by villares** [shoebot/shoebot#327]
Allison Parrish has been teaching using Flat + a helper library she made called Bezmerizing

**Comment by speters33w** [comment]
Try this:

https://github.com/shoebot/jupyter-shoebot

**Comment by villares** [comment]
> Try this:
> 
> https://github.com/shoebot/jupyter-shoebot

Yeah, I saw this, but I could't make it work on a Binder.org, I don't know the incantations to set the environment!
